KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/Server.java

```diff
@@ -175,6 +188,12 @@ public String toString() {
             public String toString() {
                 return "chunk_size"; //NON-NLS
             }
+        },
+        TERMFREQ {
```

Comment:
This doesn't appear to be a field in the schema.

Reply:
`termfreq` is a function to get the number of occurrence of the term. It's not an actual field in indices, but it can be gotten from documents in the same way as fields.
```document.getFieldValue(Server.Schema.TERMFREQ.toString())```
Should `Server.Schema` have actual fields only?

Reply:
If it's coming back as a field in the document I'm fine with leaving it where it is. I would suggest that you add a comment to avoid confusion in the future if someone is looking at the code and wondering why there is no field with that name in the schema :-)

Reply:
I added a comment on `TERMFREQ`. Thank you for your suggestion.